src/themes/material/Style.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react'
+
+export default () => (
+  <style>
+    {`
+      .material-table th span {
+        display: flex;
+        flex-wrap: nowrap;
+        align-items: center;
+        justify-content: center;
+      }
+
+
+      /* Tags Popover */
```

Comment:
if this is line for line from grey vest, I wonder if we can just import Style from there an include it 🤔